src/client/Microsoft.Identity.Client/TokenCacheNotificationArgs.cs

```diff
@@ -14,6 +14,31 @@ namespace Microsoft.Identity.Client
     /// </summary>
     public sealed partial class TokenCacheNotificationArgs
     {
+        /// <summary>
+        /// This constructor is for test purposes only. It allows apps to unit test their MSAL token cache implementation code.
+        /// </summary>
+        public TokenCacheNotificationArgs(
```

Comment:
You mean, for testing purposes for end users also, right? Or just for us internally (then it should be internal?).

Reply:
For end users, yes. All our objects should either be behind an interface (for mocking) or have a public ctor. Since this is a data object, a ctor makes sense.

This is similar to your comment a while ago on `TenantProfile`, which was not testable.